src/ProjectService.Business/Commands/Project/CreateProjectCommand.cs

```diff
@@ -226,7 +267,21 @@ private async Task CreateDepartmentEntityAsync(Guid projectId, Guid? departmentI
 
       List<Guid> imagesIds = await CreateImageAsync(request.ProjectImages, response.Errors);
 
-      DbProject dbProject = _mapper.Map(request, imagesIds);
+      List<FileData> files = request.Files.Select(x =>
+        new FileData(Guid.NewGuid(),
+          x.Name,
+          x.Content,
+          x.Extension)).ToList();
+
+      DbProject dbProject;
+      if (await CreateFileAsync(files, response.Errors))
```

Comment:
maybe here better is using ternary operator (? :)

Reply:
done